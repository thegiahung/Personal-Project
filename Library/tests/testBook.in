book 111111
book 111111 long
add book books.csv 111111
book 111111
book 111111 long
add collection books.csv
book 111112
book 111112 long
book 10
book 12 long
exit
