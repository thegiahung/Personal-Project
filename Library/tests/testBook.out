user: No books in system.

user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: To Kill a Mockingbird (Harper Lee)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

user: 16 books successfully added.

user: The Hobbit (J.R.R. Tolkien)

user: 111112: The Hobbit (J.R.R. Tolkien, Fantasy)
Currently available.

user: No such book in system.

user: No such book in system.

user: Ending Library process.
